### PR TITLE
v2 alpha prep

### PR DESCRIPTION
**Pull request by jchannon**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,4 +15,8 @@ build/
 [Oo]bj/
 [Rr]elease/
 _[Rr]e[Ss]harper.*/
-*.tss
\ No newline at end of file
+*.tss
```
Comment (thecodejunkie) [inline .gitignore]: Is this a Raider thing or what?

Comment (jchannon) [inline .gitignore]: no clue i didnt add it

File: src/Nancy.Serialization.JsonNet.MSBuild/Nancy.Serialization.JsonNet.csproj [hunk 2]
```diff
@@ -49,9 +49,9 @@
     <Compile Include="..\..\dependencies\Nancy\SharedAssemblyInfo.cs">
       <Link>Properties\SharedAssemblyInfo.cs</Link>
     </Compile>
-    <Compile Include="Helpers.cs" />
-    <Compile Include="JsonNetBodyDeserializer.cs" />
-    <Compile Include="JsonNetSerializer.cs" />
+    <Compile Include="../Nancy.Serialization.JsonNet\Helpers.cs" />
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.JsonNet.MSBuild/Nancy.Serialization.JsonNet.csproj]: These should probably be `\` for these 3

File: .gitignore (hunk 1 above)
Comment (thecodejunkie) [inline .gitignore]: Add.. indentation =/

File: src/Nancy.Serialization.JsonNet/project.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "2.0.0-*",
+
+    "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.JsonNet/project.json]: Wrong description, check the nuspec

File: src/Nancy.Serialization.JsonNet/project.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "2.0.0-*",
+
+    "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",
+    "authors": [ "Andreas Håkansson, Steven Robbins and contributors" ],
+    "tags": [ "Nancy" ],
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.JsonNet/project.json]: Wrong tags, check the nuspec

File: src/Nancy.Serialization.JsonNet/project.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    "licenseUrl": "https://github.com/NancyFx/Nancy/blob/master/license.txt",
+    "iconUrl": "http://nancyfx.org/nancy-nuget.png",
+
+    "dependencies": {
+        "Nancy": { "target": "project" },
+        "Newtonsoft.Json": "7.0.1"
+    },
+
+    "frameworks": {
+
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.JsonNet/project.json]: Blank line

File: src/Nancy.Serialization.JsonNet/project.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        "net451": {
+
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.JsonNet/project.json]: Remove the blank line and move the closing brace up to line 18

File: test/Nancy.Serialization.JsonNet.Tests/project.json [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+        "Nancy.Testing": { "target": "project" },
+        "Newtonsoft.Json": "7.0.1",
+        "xunit": "2.1.0",
+        "xunit.runner.dnx": "2.1.0-*"
+    },
+    "commands": {
+        "test": "xunit.runner.dnx"
+    },
+    "frameworks": {
+
```
Comment (thecodejunkie) [inline test/Nancy.Serialization.JsonNet.Tests/project.json]: Remove blank line

File: test/Nancy.Serialization.JsonNet.Tests/project.json [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+        "dnx451": {
+
```
Comment (thecodejunkie) [inline test/Nancy.Serialization.JsonNet.Tests/project.json]: Remove blank line and move closing brace to line 15

**Comment by thecodejunkie** [comment]
Could you add the solution folders to the `*.sln` as well please

File: rakefile.rb [hunk 9]
```diff
@@ -74,7 +75,7 @@
 
 desc "Executes xUnit tests using Mono"
 xunit :xunitmono => [] do |xunit|
```
Comment (thecodejunkie) [inline rakefile.rb]: Hmm this looks like and old task? I don't think we have that on any other RAKE-files, if not then we should kill it I think?

File: src/Nancy.Serialization.JsonNet.MSBuild/Nancy.Serialization.JsonNet.csproj (hunk 2 above)
Comment (thecodejunkie) [inline src/Nancy.Serialization.JsonNet.MSBuild/Nancy.Serialization.JsonNet.csproj]: I also think they should be links

File: test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj [hunk 10]
```diff
@@ -45,16 +48,28 @@
...
+      <HintPath>..\..\packages\xunit.extensibility.execution.2.1.0\lib\net45\xunit.execution.desktop.dll</HintPath>
       <Private>True</Private>
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="JsonNetBodyDeserializerFixture.cs" />
-    <Compile Include="JsonNetSerializerFixture.cs" />
-    <Compile Include="ModelBindingFixture.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="..\Nancy.Serialization.JsonNet.Tests\JsonNetBodyDeserializerFixture.cs" />
```
Comment (thecodejunkie) [inline test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj]: These should be links

Comment (jchannon) [inline test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj]: Links?

File: .gitignore (hunk 1 above)
Comment (jchannon) [inline .gitignore]: huh?

Comment (thecodejunkie) [inline .gitignore]: Add = ahh ;D

File: test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj (hunk 10 above)
Comment (thecodejunkie) [inline test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj]: Have a look how files are included in other `*.csproj` that have been converted https://github.com/NancyFx/Nancy/blob/master/src/Nancy.MSBuild/Nancy.csproj#L107

File: .gitignore (hunk 1 above)
Comment (jchannon) [inline .gitignore]: I think you're blind drunk, using some Swedish/English that us English people don't get or you're taking the piss. I'm going for the latter.

File: test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj (hunk 10 above)
Comment (jchannon) [inline test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj]: I'll add them in but just testing with one of those lines, it made no difference in VS

Comment (thecodejunkie) [inline test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj]: You sure it's not copying your file local when it's not treated as a link? There should be the link tag so we make sure that we only have one copy of the file for `*.csproj` and `project.json`

File: .gitignore (hunk 1 above)
Comment (thecodejunkie) [inline .gitignore]: This is a more accurate representation

image

File: test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj [hunk 11]
```diff
@@ -45,16 +48,36 @@
...
+      <Link>JsonNetBodyDeserializerFixture.cs</Link>
+    </Compile>
+    <Compile Include="..\Nancy.Serialization.JsonNet.Tests\JsonNetSerializerFixture.cs">
+      <Link>JsonNetSerializerFixture.cs</Link>
+    </Compile>
+    <Compile Include="..\Nancy.Serialization.JsonNet.Tests\ModelBindingFixture.cs">
+      <Link>ModelBindingFixture.cs</Link>
+    </Compile>
+    <Compile Include="..\Nancy.Serialization.JsonNet.Tests\Properties\AssemblyInfo.cs">
+    <Link>Properties\AssemblyInfo.cs</Link>
```
Comment (thecodejunkie) [inline test/Nancy.Serialization.JsonNet.Tests.MSBuild/Nancy.Serialization.JsonNet.Tests.csproj]: Indentation

File: src/Nancy.Serialization.JsonNet.MSBuild/Nancy.Serialization.JsonNet.csproj [hunk 12]
```diff
@@ -49,9 +49,9 @@
     <Compile Include="..\..\dependencies\Nancy\SharedAssemblyInfo.cs">
       <Link>Properties\SharedAssemblyInfo.cs</Link>
     </Compile>
-    <Compile Include="Helpers.cs" />
-    <Compile Include="JsonNetBodyDeserializer.cs" />
-    <Compile Include="JsonNetSerializer.cs" />
+    <Compile Include="..\Nancy.Serialization.JsonNet\Helpers.cs" />
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.JsonNet.MSBuild/Nancy.Serialization.JsonNet.csproj]: Should be links

**Pull request merged**